### DISCUSSION TITLE
Need help finding a PHP/Laravel solution for my project

### DISCUSSION THREAD

**Discussion by AZNYAZNY** [community/community#37583] [Other Feature Feedback, Questions, & Ideas]
Can someone help us find existing OPEN SOURCE, PHP/LARAVEL based software that is easy to customize and stable and that will lost long-term to manipulate and customize for our new service.

The service being launched is a marketplace that has a "provider" and a "consumer".  It is very similar to something like GROUPON as far as functionality in that users can "create" packages and consumers can purchase those packages.

**Comment by ossanna16** [comment]
Hi @AZNYAZNY,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on PHP and Laravel. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
